apps/condo/domains/common/components/icons/GearIcon.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import Icon from '@ant-design/icons'
+
+const GearIconSVG: React.FC = () => {
+    return (
+        <svg width="20" height="20" viewBox="0 0 20 22" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
May be better to grab width & height attributes from component props instead of make it constant value

Reply:
is it common for all our icons?

Reply:
It was...before "Analytics section added"

Reply:
resolved 👍